shell/platform/windows/flutter_windows.cc

```diff
@@ -246,6 +246,13 @@ bool FlutterDesktopEngineProcessExternalWindowMessage(
   return lresult.has_value();
 }
 
+void FlutterDesktopEngineRegisterPlatformViewType(
+    FlutterDesktopEngineRef engine,
+    const char* view_type_name,
+    FlutterPlatformViewTypeEntry view_type) {
+  // TODO(schectman): forward to platform view manager
```

Comment:
Don't forget a link to tracking issue: https://github.com/flutter/flutter/issues/138431